tests/mapped_struct.py

```diff
@@ -1314,10 +1321,10 @@ class BsearchTest(unittest.TestCase):
 
     for dtype in SUPPORTED_DTYPES:
         def testBsearch(self, dtype=dtype):
-            testarray = range(1,101)
+            testarray = list(range(1,101))
             random.shuffle(testarray)
             a = numpy.array(testarray[:50], dtype)
-            b = numpy.array([0] + testarray[50:] + range(101,103), dtype)
+            b = numpy.array([0] + testarray[50:] + list(range(101,103)), dtype)
```

Comment:
``` python
from past.builtins import range, xrange
```

And then you don't have to change these and it works as expected in all cases.

FTR, past and future helpers are only good for tests and pure python code. Cythonized code can't benefit from these because the wrappers aren't recognized in optimizations, so it generates horrible code for their use.